assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -100,6 +100,13 @@ angular.module("openshiftConsole")
             scope.buildConfig.contextDir = annotations.sampleContextDir || "";
           };
 
+          scope.usingSampleRepo = function() {
+            if (!scope.image && !scope.image.metadata && !scope.image.metadata.annotations) {
+              return false;
+            }
+            return scope.buildConfig.sourceUrl === scope.image.metadata.annotations.sampleRepo;
```

Comment:
Fixed to use `_.get`, but I still have to compare it to the `sourceUrl` in the bc so I know I'm using the same specified as sample repo for that image, correct?